dsc/tests/dsc_expressions.tests.ps1

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'Expressions tests' {
+    It 'Accessors work: <text>' -TestCases @(
+        @{ text = "[parameters('test').hello]"; expected = '@{world=there}' }
+        @{ text = "[parameters('test').hello.world]"; expected = 'there' }
```

Comment:
I see this covers the member accessor case - is the index accessor case also covered?

Reply:
good catch, looks like I setup the parameters for it, but forgot to add in the test cases